eosio/bbs.env

```diff
@@ -1,3 +1,11 @@
 # Environment variables for BBS token on the eosio side.
 nodeos=https://kylin-dsp-1.liquidapps.io
-faucet=http://faucet-kylin.blockzone.net
\ No newline at end of file
+faucet=http://faucet-kylin.blockzone.net
+
+#EOS_BBS_TOKEN_MAXIMUM_SUPPLY="'1000000.0000 BBS'"
+#EOS_BBS_TOKEN_ISSUE_AMOUNT="100.0000 BBS"
+#EOS_BRIDGE_MIN_REPORTERS='1'
+#EOS_BRIDGE_MIN_LIMIT='10000'
+#EOS_BRIDGE_LIMIT_INC='10000'
+#EOS_BRIDGE_MAX_ISSUE_LIMIT='800000000'
+#EOS_BRIDGE_MAX_DESTROY_LIMIT='400000000'
```

Comment:
You said this doesn't work, and IIRC the reason for this is that you do not source `env.sh` in the script that tries to use them. Please check if this is the case, and fix accordingly.

Reply:
solved by separating "100.0000 BBS" to two variables.